gensim/summarization/graph.py

```diff
@@ -303,31 +289,27 @@ def add_edge(self, edge, wt=1, label='', attrs=None):
             Given edge.
         wt : float, optional
             Weight of new edge.
-        label : str, optional
-            Edge label.
-        attrs : list, optional
-            Node attributes specified as (attribute, value).
 
         Raises
         ------
         ValueError
             If `edge` already exists in graph.
 
         """
-        if attrs is None:
-            attrs = []
+        if wt == 0.0:
+            # empty edge is similar to no edge at all or removing it
+            if self.has_edge(edge):
+                self.del_edge(edge)
+            return
         u, v = edge
         if v not in self.node_neighbors[u] and u not in self.node_neighbors[v]:
-            self.node_neighbors[u].append(v)
+            self.node_neighbors[u][v] = wt
```

Comment:
How looks `node_neighbors` now, is it 
```python
{"node_1": {"node_25": 0.5, ...} ...}

```
?

Reply:
Yes, you're right. But one nuance: nodes actually are bow-s. It's a documents from corpus.

Reply:
that's fine, thanks for clarification